### ISSUE TITLE
Firejail cannot handle special characters, shutsdown

### ISSUE THREAD

**Issue by g4jc** [netblue30/firejail#207]
This bug only effects files with special characters, such as ()?!.
Perhaps there is some way to sanatize the input prior to starting the chroot.

Programs affected: Image viewers, PDF viewers, some media players.
```
user@localhost ~/Downloads % feh merry-christmas-quotes-\(2\).jpg 
user@localhost ~/Downloads % echo "the above command works fine... now let's try to firejail it..."
user@localhost ~/Downloads % firejail feh merry-christmas-quotes-\(2\).jpg
Reading profile /etc/firejail/generic.profile
Reading profile /etc/firejail/disable-mgmt.inc
Reading profile /etc/firejail/disable-secret.inc
Reading profile /etc/firejail/disable-common.inc

** Note: you can use --noprofile to disable generic.profile **

Parent pid 3764, child pid 3765
Warning: /sbin directory link was not blacklisted
Warning: /usr/sbin directory link was not blacklisted
Warning: cannot disable /sys/hypervisor directory
Child process initialized
/bin/bash: -c: line 0: syntax error near unexpected token `('
/bin/bash: -c: line 0: `feh merry-christmas-quotes-(2).jpg '
parent is shutting down, bye...
```

**Comment by netblue30** [comment]
I'll have to look into it, thanks!

**Comment by netblue30** [comment]
It looks like this was fixed in the latest version, 0.9.36

**Comment by ivan** [comment]
Thanks for the fix, I can finally stop double-quoting everything :smiley:
